phone/src/apps/match/hooks/useMatchNotifications.ts

```diff
@@ -0,0 +1,42 @@
+import { useTranslation } from 'react-i18next';
+import { matchPath, useHistory } from 'react-router-dom';
+import { useApp } from '@os/apps/hooks/useApps';
+import { useNotifications } from '@os/notifications/hooks/useNotifications';
+
+const NOTIFICATION_ID = 'match:broadcast';
+
+export const useMatchNotifications = () => {
+  const [t] = useTranslation();
+  const history = useHistory();
+  const { removeId, addNotification, addNotificationAlert } = useNotifications();
+  const { icon, notificationIcon } = useApp('MATCH');
+
+  history.listen((location) => {
+    if (
+      matchPath(location.pathname, {
+        path: `/match/matches`,
+        exact: true,
+      })
+    ) {
+      removeId(NOTIFICATION_ID);
+    }
+  });
```

Comment:
React hooks are a form of functional programming. By not wrapping this in a `useEffect` this would create an assignment with every rerender of a consumer.

Reply:
Any dependency?